questions/lookat.md

```diff
@@ -27,6 +27,68 @@ Example 2: looking at the position 10 20 30
 
     execute as @a at @s anchored eyes facing 10 20 30 anchored feet positioned ^ ^ ^1 rotated as @s positioned ^ ^ ^-1 if entity @s[distance=..0.1] run say hello block
 
+## Java
+
+### Predicate
+There is a predicate that allows us to detect when a player is looking at an entity, it's the one used by the 3 advancements related to the spyglass.
+
+    # function example:tick
+    execute as @a[predicate=example:looking_cow] run say Hi, cow!
+    
+
+    # predicate example:looking_cow
+    {
+      "condition": "minecraft:entity_properties",
+      "entity": "this",
+      "predicate": {
+        "type_specific": {
+          "type": "minecraft:player",
+         "looking_at": {
+            "type": "minecraft:cow"
+          }
+        }
+      }
+    }
+
+### Advancement
+This example is the same advancement as the vanilla ones but without any display.
+You can find (and edit) the preset in [misode](https://misode.github.io/advancement/?version=1.20.5&preset=adventure/spyglass_at_ghast))
```

Comment:
"in misode" sounds weird - misode is the person who made it, so it would be better to write "in misode's advancement generator"